server/models/User.js

```diff
@@ -0,0 +1,72 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema(
+  {
+  // =====================================
+  //  retrieved from google login response
+  // =====================================
+    name: {
+      type: String,
+      minlength: 1,
+      maxlength: 30,
+      trim: true,
+      required: true,
+    },
+    email: {
+      type: String,
+      minlength: 1,
+      maxlength: 30,
+      trim: true,
+      required: true,
+      unique: true
+    },
+    avatarUrl:{
+      type:String,
+      required:true
+    },
+    // =====================================
+    //created by user during onboarding
+    // =====================================
+    timeZone: {
+      type: String,
+      enum: [
+        "UTC-3 ADT",
+        "UTC-4 EDT",
+        "UTC-5 CDT",
+        "UTC-6 MDT",
+        "UTC-7 PDT"
+      ],
+      required: true,
+    },
+    //not decided on available hour yet
+    availableHours: {
```

Comment:
I'm wondering if we should do `startAvailableHour` and `endAvailableHour` instead